src/ChatWindow.js

```diff
@@ -167,9 +160,12 @@ class ChatWindow extends Component {
     // If authenticationInfo is defined, then access_token is defined.
     if (this.props.authenticationInfo) {
       access_token = this.props.authenticationInfo.getAccessToken();
+    } else {
+      // Do not make a call to avs if access_token is undefined. Redirect the user to login screen.
+      this.props.clearAuthenticationInfo();
+      return false;
```

Comment:
Does this have any significance? What happens if I return true here?

Reply:
This has no significance in the current infrastructure. Return values are not being used. 

EventHandlers were supposed to return false if desired functionality is to preventDefault() and stopPropagation(). From React 0.12, this was deprecated.

From https://github.com/reagent-project/reagent/issues/78

- when you return false in react 0.11, it calls BOTH stopPropagation() and preventDefault().
- when you return false in react 0.12, it does the same thing BUT it spits out the deprecation warning.
- in react 0.13 the return value may be used for a completely different purpose